android/assets/jsons/Units.json

```diff
@@ -530,7 +530,7 @@
 		"strength": 16,
 		"cost": 45,
 		"requiredTech": "Civil Service",
-		"uniques": ["Bonus vs Mounted 50%", "Starts with no movement penalty" ],
+		"uniques": ["Bonus vs Mounted 50%", "Can move directly once bought" ],
```

Comment:
Why did you decide to change the wording? The initial one seems to be better.
I understand "directly" like "straight". Maybe, "immediately" fits better?

Reply:
I changed it cause I felt like the meaning of "movement penalty" is not immediately clear esp if u just read the description. Additionally it was kind of clumsy to translate into german.